redirects.yaml

```diff
@@ -911,3 +911,12 @@ nvidia: /engage/ubuntu-at-nvidia-gtc-2021
 # Move /securtiy/cve to /security/cves
 security/cve: /security/cves
 security/cve/sitemap(?P<suffix>.*).xml: /security/cves/sitemap{suffix}.xml
+
+# Shop redirects
+/advantage/purchase-account: "/account/purchase-account"
```

Comment:
I don't think the redirects on "POST" endpoints will work.

Reply:
I can re-QA it. But it think it worked.